{"uuid": "b8fc9775-a509-4078-a8b4-ff8778895bc4", "children": ["5a24a96c-bd57-4099-997b-7c904225d2c8"], "befores": [{"name": "_xunit_setup_class_fixture_TestWeCom", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 10.000\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n\tGetHandleVerifier [0x00007FF6EA71FDA5+29557]\n\t(No symbol) [0x00007FF6EA692240]\n\t(No symbol) [0x00007FF6EA54B6EA]\n\t(No symbol) [0x00007FF6EA5341C7]\n\t(No symbol) [0x00007FF6EA533ECA]\n\t(No symbol) [0x00007FF6EA531D52]\n\t(No symbol) [0x00007FF6EA5325AF]\n\t(No symbol) [0x00007FF6EA55A100]\n\t(No symbol) [0x00007FF6EA5F7E9A]\n\t(No symbol) [0x00007FF6EA5C74FA]\n\t(No symbol) [0x00007FF6EA5E88A3]\n\t(No symbol) [0x00007FF6EA5C72A3]\n\t(No symbol) [0x00007FF6EA5912DF]\n\t(No symbol) [0x00007FF6EA592451]\n\tGetHandleVerifier [0x00007FF6EAA4DCBD+3363469]\n\tGetHandleVerifier [0x00007FF6EAA99B47+3674391]\n\tGetHandleVerifier [0x00007FF6EAA8EAEB+3629243]\n\tGetHandleVerifier [0x00007FF6EA7DFC66+815670]\n\t(No symbol) [0x00007FF6EA69D6EF]\n\t(No symbol) [0x00007FF6EA6992B4]\n\t(No symbol) [0x00007FF6EA699450]\n\t(No symbol) [0x00007FF6EA6881FF]\n\tBaseThreadInitThunk [0x00007FFD36C4257D+29]\n\tRtlUserThreadStart [0x00007FFD3884AF08+40]\n\n", "trace": "  File \"D:\\python3\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\python.py\", line 787, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, cls)\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\python.py\", line 715, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\开发项目\\selenium_test_WeCom\\test_case\\test_wecom.py\", line 30, in setup_class\n    self.home = LoginPage().reuse_cookies_login()\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\开发项目\\selenium_test_WeCom\\page_obj\\login_page.py\", line 25, in reuse_cookies_login\n    self.get_screen_shot()\n  File \"D:\\开发项目\\selenium_test_WeCom\\page_obj\\base_page.py\", line 99, in get_screen_shot\n    self.driver.save_screenshot(image_path)\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 822, in save_screenshot\n    return self.get_screenshot_as_file(filename)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 798, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 832, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 843, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "前置条件：企业微信复用cookie登录成功，并截图和打印page_source", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 10.000\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n\tGetHandleVerifier [0x00007FF6EA71FDA5+29557]\n\t(No symbol) [0x00007FF6EA692240]\n\t(No symbol) [0x00007FF6EA54B6EA]\n\t(No symbol) [0x00007FF6EA5341C7]\n\t(No symbol) [0x00007FF6EA533ECA]\n\t(No symbol) [0x00007FF6EA531D52]\n\t(No symbol) [0x00007FF6EA5325AF]\n\t(No symbol) [0x00007FF6EA55A100]\n\t(No symbol) [0x00007FF6EA5F7E9A]\n\t(No symbol) [0x00007FF6EA5C74FA]\n\t(No symbol) [0x00007FF6EA5E88A3]\n\t(No symbol) [0x00007FF6EA5C72A3]\n\t(No symbol) [0x00007FF6EA5912DF]\n\t(No symbol) [0x00007FF6EA592451]\n\tGetHandleVerifier [0x00007FF6EAA4DCBD+3363469]\n\tGetHandleVerifier [0x00007FF6EAA99B47+3674391]\n\tGetHandleVerifier [0x00007FF6EAA8EAEB+3629243]\n\tGetHandleVerifier [0x00007FF6EA7DFC66+815670]\n\t(No symbol) [0x00007FF6EA69D6EF]\n\t(No symbol) [0x00007FF6EA6992B4]\n\t(No symbol) [0x00007FF6EA699450]\n\t(No symbol) [0x00007FF6EA6881FF]\n\tBaseThreadInitThunk [0x00007FFD36C4257D+29]\n\tRtlUserThreadStart [0x00007FFD3884AF08+40]\n\n", "trace": "  File \"D:\\开发项目\\selenium_test_WeCom\\page_obj\\login_page.py\", line 25, in reuse_cookies_login\n    self.get_screen_shot()\n  File \"D:\\开发项目\\selenium_test_WeCom\\page_obj\\base_page.py\", line 99, in get_screen_shot\n    self.driver.save_screenshot(image_path)\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 822, in save_screenshot\n    return self.get_screenshot_as_file(filename)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 798, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 832, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 843, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\python3\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1729265088156, "stop": 1729265151842}], "start": 1729265073151, "stop": 1729265151843}], "start": 1729265073151, "stop": 1729265151904}